app/controllers/events_controller.rb

```diff
@@ -59,16 +59,11 @@ def rsvp
 
   def find_invitation_and_redirect_to_event(role)
     set_event
-    @invitation = Invitation.where(event: @event, member: current_user, role: role).try(:first)
-    if @invitation.nil?
-      @invitation = Invitation.new(event: @event, member: current_user, role: role)
-      @invitation.save
-    end
-
+    @invitation = Invitation.find_or_create_by(event: @event, member: current_user, role: role)
```

Comment:
@matyikriszta coverage dropped because I shrunk this method. Am I ok to get a merge despite this github check failing?